doc/page-cmakefunctions.h

```diff
@@ -122,4 +122,17 @@ This function adds a test with the name \e TargetName.The parameter descriptions
 \param URI The URI of the Target i.e qml plugin
 \param VERSION The target i.e Qml plugin version, if not provided it defaults to 1.0
 
+\section cmake-params CMake options
```

Comment:
This page describes the CMake functions/macros which Facelift provides to any package.
The configure options do not really belong here. I would rather have them in a separate "configuration options" section